RealmSwift/RealmCollection.swift

```diff
@@ -1053,22 +1055,78 @@ public struct AnyRealmCollection<Element: RealmCollectionValue>: RealmCollection
      // end of run loop execution context
      ```
 
+     If no key paths are given, the block will be executed on any insertion,
+     modification, or deletion for all collection type properties and nested linked
+     properties (up to a depth of 4). If a key path or key paths are provided,
```

Comment:
Same as for `Aliases`.

Reply:
Resolved in [b8e9ee0](https://github.com/realm/realm-cocoa/pull/7249/commits/b8e9ee02afed35f06f77bd688886ad6b7aea627d)